AdyenCheckout/AdyenCheckoutComponent.swift

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+#if canImport(AdyenSession)
+    @_spi(AdyenInternal) import AdyenSession
+#endif
+#if canImport(AdyenDropIn)
+    @_spi(AdyenInternal) import AdyenDropIn
```

Comment:
Side thought: Now `AdyenCheckout` is an "umbrella" module that holds references to all other references. 

This means DropIn no longer need to have `ComponentBuilder` and it can become a fancy UI tool with thin business login, but no payment logic whatsoever 🤔

Reply:
Yes indeed! Beauty of this is we can change the internal stuff later